client/scss/components/_preview-panel.scss

```
@@ -3,6 +3,7 @@
   display: flex;
   flex-direction: column;
 
+  --color-white: #fff;
```

I know the existing variables do not have this convention but let's align with the current preferred naming and use the existing tokens. https://docs.wagtail.org/en/stable/contributing/ui_guidelines.html#css-guidelines

```suggestion
  --w-preview-background-color: var(--w-color-white);
```

While this will not be documented, it lets this be easily overridden and avoids conflicts with other global names.

No need to update the others though, that may be a bigger change (a good second PR if you wanted), but will need some changes in JS.

---

Thanks for the suggestions and the convention tip here @lb-.  

I think a white-bg-when-empty approach goes against the concern of #11174; A new dev (or someone working on a new site) might not have set a bg color for the previewed document. The white background of the iframe is expected to be replaced by the actual bg color of the previewed page; If no background were set for a page in any modern browser a white background would be automatically shown. A div with a warning about iframes not being supported wouldn't be visible with the dark theme and no background.

Going off your comment above, I think it'd make sense to not set a background for the iframe if empty to avoid the potential of a flash while loading the preview (and have updated the PR to reflect this).

---

It's best if you test out the approach I've suggested. 

What happens is that the background gets set when the IFrame loads. Because the initial DOM gets replaced with the IFrame content and this is considered 'empty' when the CSS selectors run. 

So by using `empty` we are actually saying - when the IFrame loads. It seems counter intuitive but makes sense when you think of the loaded IFrame content as not real DOM nodes. 

https://drafts.csswg.org/selectors/#the-empty-pseudo
https://stackoverflow.com/a/61764581/8070948

---

This will also require the added `div` in the preview html. Otherwise it will show the background before the IFrame is loaded still.

---

👍 Tested that, looks good. Was reading your comment a bit differently. This should be ready to go.

---

Tests passing, thanks all